docs/components/modeler/desktop-modeler/index.md

```diff
@@ -23,6 +23,8 @@ Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms and s
 
 Download the app for Windows, Linux, or macOS from the [Camunda downloads page](https://camunda.com/download/modeler/).
 
+> On Windows and Linux you can carry out [additional steps](./install-the-modeler#wire-file-associations) to register the modeler as the default editor for BPMN, CMMN and DMN files.
```

Comment:
Let's mention Form files as they are part of [the tool core functionality](https://github.com/camunda/camunda-modeler/?tab=readme-ov-file#camunda-modeler). We can omit CMMN as a legacy feature. 

```suggestion
> On Windows and Linux you can carry out [additional steps](./install-the-modeler#wire-file-associations) to register the modeler as the default editor for BPMN, DMN, and Form files.
```